### ISSUE TITLE
to_string short currency format with decimals

### ISSUE THREAD

**Issue by glennr** [elixir-cldr/cldr_numbers#32]
Hi!

When upgrading from v2.15 to v2.27.1 I noticed this regression with :short + :currency and decimals.

Floats are fine:

```
iex(40)> Cldr.Number.to_string(219.50, format: :short, currency: "USD", fractional_digits: 2)
{:ok, "$219.50"}
```

But decimal rounding seems wrong:

```
iex(40)> Cldr.Number.to_string(Decimal.from_float(219.50), format: :short, currency: "USD", fractional_digits: 2)
{:ok, "$219.00"}
```

Expected: `{:ok, "$219.50"}`

Versions Im using:

:decimal, "2.0.0"
:ex_cldr, "2.31.0"
:ex_cldr_numbers, "2.27.1"

**Comment by kipcole9** [comment]
@glennr, on it, fix is being tested, should have a release out within 30m.

**Comment by kipcole9** [comment]
@glennr, thanks much again your patience and the report. Fixed and published as [ex_cldr_numbers version 2.27.2](https://hex.pm/packages/ex_cldr_numbers/2.27.2). The changelog entry reads:

### Bug Fixes

* Fix `:short` formatting of decimal numbers with specified fractional digits. Thanks to @glennr for the report. Closes #32.

**Comment by glennr** [comment]
Legend. Thanks for the super quick turnaround again!
